initialize_language.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+set -e
+
+if [ $# -ne 1 ]
+then
+	echo "usage: $0 language_code"
+	exit 0
+fi
+
+IROHA_LANG="$1"
+IROHA_LANG_DIR="$PWD/docs/locale/$IROHA_LANG"
+IROHA_DOCS_DIR="$IROHA_LANG_DIR/docs"
+
+# Prepare folder for translation
+mkdir -p "$IROHA_DOCS_DIR"
+
+# Create symlinks for docs directory
+lndir "$PWD"/iroha/docs/ "$IROHA_DOCS_DIR"
+
+# Replace absolute symlinks with relative
+symlinks -rc "$IROHA_LANG_DIR"
+
+# Replace symlinks with file copies for rst, md, csv, yaml
+for f in $(find "$IROHA_LANG_DIR" -type l \( -name '*.rst' -o -name '*.md' -o -name '*.csv' -o -name '*.yaml' \))
+do
+	cp --remove-destination $(readlink -f $f) $f
+done
+
+# Copy CONTRIBUTING.md from iroha repo root
+cp "$PWD"/iroha/CONTRIBUTING.md "$IROHA_LANG_DIR"/
```

Comment:
```suggestion
# Copy CONTRIBUTING.rst from iroha repo root
cp "$PWD"/iroha/CONTRIBUTING.rst "$IROHA_LANG_DIR"/
```
Current master and 1.2.x have rst instead of md. We will have to modify the script depending on Iroha version.